cluster/cluster.go

```diff
@@ -482,7 +451,7 @@ func DeleteTenantFolderInDisk(tenant *Tenant, sg *StorageGroup, sgNode *StorageG
 
 			newFolderForDeletion := fmt.Sprintf("%s/%s-to-delete-%s", vol.MountPath, tenant.ShortName, randSringForDeletion)
```

Comment:
it is perhaps better to use a much more unique name so that avoids `mv` conflict perhaps stick a uuid into this directory.

Reply:
done